group_vars/figgy/staging.yml

```diff
@@ -135,9 +135,10 @@ rails_app_vars:
   - name: PULFALIGHT_UNPUBLISHED_TOKEN
     value: '{{ vault_pulfalight_unpublished_auth_token }}'
 sidekiq_worker_name: figgy-workers
-passenger_ruby: "/usr/bin/ruby2.7"
-ruby_version_override: "ruby2.7"
-bundler_version: "2.3.18"
+passenger_ruby: "/usr/local/bin/ruby"
+install_ruby_from_source: true
+ruby_version_override: "ruby-3.1.0"
```

Comment:
figgy doesn't run on ruby 3 yet. it's a high priority but not finished. is it a blocker for this work?

Reply:
We should be able to install ruby 2.7.5 from source. +1 to separating the ruby upgrade from the OS upgrade.